src/crypto.rs

```diff
@@ -131,6 +131,8 @@ fn shim_public_key(
     signature_scheme: &SignatureScheme,
     keyid_hash_algorithms: &Option<Vec<String>>,
     public_key: &[u8],
+    private_key: bool,
```

Comment:
Checking: this is to be compatible with python that keeps a privkey field that's empty for pubkeys right?

Reply:
Yes. The parameter `bool` here refers to what python version does.